pkg/internal/token/interactive.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
 	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
 	"github.com/Azure/go-autorest/autorest/adal"
-	"github.com/Azure/kubelogin/pkg/internal/pop"
+	"github.com/Azure/kubelogin/pkg/pop"
```

Comment:
we cannot let internal package to import a public package as this is might be troublesome when there is cyclic imports and api changes.